network-api/networkapi/wagtailpages/pagemodels/blog/blog_category.py

```diff
@@ -6,6 +6,8 @@
 from wagtail.core.models import TranslatableMixin
 from wagtail.snippets.models import register_snippet
 
+from networkapi.wagtailpages.pagemodels.customblocks.base_rich_text_options import base_rich_text_options
+
```

Comment:
```suggestion
from django.db.utils import ProgrammingError
```
Although we probably want this directly under the other db import

Reply:
Added to latest commit! Marking as resolved

Reply:
Added to latest commit! Marking as resolved